doc/source/comparison_with_sas.rst

```diff
@@ -14,8 +14,8 @@ As is customary, we import pandas and NumPy as follows:
 
 .. ipython:: python
 
-    import pandas as pd
-    import numpy as np
+    import pandas as pd    # noqa: F811
+    import numpy as np     # noqa: F811
```

Comment:
why don't we use the header here? you can add the `F811` to the `--bootstrap` parameter value if needed

Reply:
This is a tutorial of sorts and the tutorial walks thru the process of importing pandas and numpy visibly (not with a :supress: ). I think without the visible import of pandas the tutorial will not make sense.

Reply:
Makes sense. But can you still include the header in those files? Besides the imports, there is other stuff. And we will potentially add more, and expect to be applied to 100% of rst files. Thanks!